### PR TITLE
PCQ-1904 test cases added

### PR DESCRIPTION
**Pull request by priyankaVerma21**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/PCQ-1904

### Change description ###
Test cases added for new code

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/controllers/reviewController.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export class ReviewController {
           const url = new URL(UrlRoute.make(req.session.callbackUrl, { id: '' }));
           res.set('Content-Security-Policy', `form-action 'self' ${url}`);
         } catch (err) {
-          throw new Error(ErrorMessages.UNEXPECTED_ERROR + ':' + err);
+          throw new Error(ErrorMessages.UNEXPECTED_ERROR + err);
```
Comment (MartinYSpasov) [inline src/main/controllers/reviewController.ts:25]: UNEXPECTED_ERROR constant hasn't got colon, Why did you remove this?

File: src/test/unit/tests/controllers/reviewController.test.ts [hunk 2]
```diff
@@ -72,6 +72,27 @@ describe('Review Controller', () => {
     expect(mockResponse.render).toBeCalledWith('review', expect.any(Object));
   });
 
+  test('Should not render review page', async () => {
+    // eslint-disable-line @typescript-eslint/no-empty-function
```
Comment (MartinYSpasov) [inline src/test/unit/tests/controllers/reviewController.test.ts:76]: do we need this?

File: src/test/unit/tests/controllers/reviewController.test.ts [hunk 3]
```diff
@@ -229,4 +250,26 @@ describe('Review Controller', () => {
...
+      } as unknown as Session & Partial<SessionData>,
+      protocol: protocol,
+      headers: {
+        host: host,
+      },
+    };
+    await reviewController.post(mockRequest as Request, mockResponse as Response, mockNext);
+    expect(mockNext).toBeCalledWith(new Error(ErrorMessages.UNEXPECTED_ERROR + ErrorMessages.INVALID_URL));
+
+    //expect(mockResponse.redirect).toBeCalledWith(new URL('https://localhost/callback/random-string-uuid'));
```
Comment (MartinYSpasov) [inline src/test/unit/tests/controllers/reviewController.test.ts:273]: Do we need this commented code?

File: src/test/unit/tests/controllers/reviewController.test.ts [hunk 4]
```diff
@@ -229,4 +250,26 @@ describe('Review Controller', () => {
 
     expect(mockResponse.redirect).toBeCalledWith(new URL('https://localhost/callback/random-string-uuid'));
   });
+
+  test('Should submit review and fail to callback', async () => {
+    // eslint-disable-line @typescript-eslint/no-empty-function
```
Comment (MartinYSpasov) [inline src/test/unit/tests/controllers/reviewController.test.ts:255]: Do we need this commented code?

**Review by MartinYSpasov — CHANGES_REQUESTED** [review]

File: src/main/controllers/reviewController.ts (hunk 1 above)
Comment (priyankaVerma21) [reply src/main/controllers/reviewController.ts:25]: I have only added this colon and Unexpected error has "." so there is no point to have ".:" together so removed it

**Review by MartinYSpasov — APPROVED** [review]
